modules/tensor_mechanics/src/materials/ComputeStressBase.C

```diff
@@ -87,3 +84,12 @@ ComputeStressBase::computeQpProperties()
   // Add in extra stress
   _stress[_qp] += _extra_stress[_qp];
 }
+
+void
+ComputeStressBase::addQpInitialStress(RankTwoTensor & stress)
+{
+  if (_initial_stress.size() == 3 * 3)
+    for (unsigned i = 0; i < 3; ++i)
+      for (unsigned j = 0; j < 3; ++j)
+        stress(i, j) += _initial_stress[i * 3 + j]->value(_t, _q_point[_qp]);
```

Comment:
I see that RankTwoTensor has a private static constexpr unsigned N=LIBMESH_DIM.  So we should use LIBMESH_DIM instead of 3, for consistency.  Perhaps we could make that "N" parameter public and use that instead, for ease of changing later on.  But that's for another PR.